lib/generators/rspec/authentication/authentication_generator.rb

```diff
@@ -0,0 +1,25 @@
+require 'generators/rspec'
+
+module Rspec
+  module Generators
+    # @private
+    class AuthenticationGenerator < Base
+      def initialize(args, *options)
+        args.replace(['User'])
+        super
+      end
+
+      def create_user_spec
+        template 'user_spec.rb', target_path('models', 'user_spec.rb')
+      end
+
+      hook_for :fixture_replacement
```

Comment:
I added this to mimic the model generator, but maybe this not desired as that means extra work for fixture replacement gems like factory bot. I made https://github.com/thoughtbot/factory_bot_rails/pull/519.